lib/scanner/index.js

```diff
@@ -202,11 +165,20 @@ class Scanner {
       .filter(key => !!this.items[key].data)
       .map((key) => this.items[key].data);
 
-    this.callback(items);
+    this.emit('update', items);
```

Comment:
Now using an Emitter instead of callbacks. This allows multiple listeners, which makes testing easier.